.github/workflows/fast-dispatch-full-regressions-and-models.yaml

```diff
@@ -103,12 +103,19 @@ jobs:
               cmd: ./tests/scripts/run_tests.sh --tt-arch wormhole_b0 --pipeline-type frequent_api --dispatch-mode fast,
               timeout: 40
             },
+            {
+              name: "[Unstable] N150 models",
+              arch: wormhole_b0,
+              runs-on: ["cloud-virtual-machine", "arch-wormhole_b0", "N150", "in-service", "mount-cloud-weka"],
+              cmd: tests/scripts/single_card/nightly/run_wh_b0_unstable.sh,
+              timeout: 55
```

Comment:
Are we adding this intentionally for patch missing test coverage of unstable wormhole?

Reply:
Yes, @esmalTT requested in ticket #11370